src/enhetsportefolje/enhet-listehode.tsx

```diff
@@ -214,8 +213,8 @@ function EnhetListehode({
                     onClick={sorteringOnClick}
                     rekkefolge={sorteringsrekkefolge}
                     erValgt={sorteringsfelt === Sorteringsfelt.VALGTE_AKTIVITETER}
-                    tekst="Neste utløpsdato aktivitet"
-                    skalVises={avansertAktivitet || forenkletAktivitet}
+                    tekst="Neste utløpsdato tiltak"
```

Comment:
Er det faktisk slik at det er neste utløpsdato for tiltak?

Reply:
Ja, det er korrekt!